docs/development/documentation_guidelines.rst

```diff
@@ -3,12 +3,28 @@ Documentation Guidelines
 
 SecureDrop's documentation is written in `ReStructuredText`_ (ReST),
 and is built by and hosted on `Read the Docs`_ (RTD). The
-documentation files are stored in the primary SecureDrop git
-repository under the ``docs/`` directory.
+documentation files are stored in the ``docs/`` directory of the `SecureDrop docs repository <https://github.com/freedomofpress/securedrop-docs>`_.
 
 .. _ReStructuredText: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
 .. _Read the Docs: https://docs.readthedocs.org/en/latest/index.html
 
+
+Integration with Read the Docs
+------------------------------
+
+.. include:: ../includes/docs-branches.txt
+
+Our documentation is built and hosted by `Read the Docs`_ and is available at
+https://docs.securedrop.org. We use a `webhook`_ to rebuild the documentation
+automatically when commits get pushed to the branch.
```

Comment:
Moving the paragraph that starts with "Our documentation .." makes it obvious how duplicative it is -- compare it with the first paragraph in this chapter:

> SecureDrop’s documentation is written in ReStructuredText (ReST), and is built by and hosted on Read the Docs (RTD). The documentation files are stored in the primary SecureDrop git repository under the docs/ directory.

I would suggest consolidating these two paragraphs in the introductory paragraph. The use of a webhook is a technical detail that I would be happy to remove.